### PR TITLE
fix(security): update path-to-regex

### PR DESCRIPTION
**Pull request by donal-tobin-sap**
- also related parent module express
- and related module body parser
- dependencies are dev only

Issue: tools-suite/issues/31259
Issue: https://github.com/SAP/open-ux-tools/security/dependabot/148
Issue: https://github.com/SAP/open-ux-tools/security/dependabot/149
Issue: https://github.com/SAP/open-ux-tools/security/dependabot/135

### CODE REVIEW

**Review by IainSAP — COMMENTED** [review]
Accidental test file change

File: packages/odata-service-inquirer/test/unit/prompts/cap-project/fixtures/bookshop/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@capire/common": "../common",
     "@sap/cds": ">=3.33.1",
-    "express": "^4.19.2"
+    "express": "^4.20.0"
```
[the remark this answers is not in the document]
Comment (donal-tobin-sap) [reply packages/odata-service-inquirer/test/unit/prompts/cap-project/fixtures/bookshop/package.json]: Reverted

**Review by IainSAP — APPROVED** [review]
Thanks @donal-tobin-sap 

- Security updates as expected

**Review by longieirl — DISMISSED** [review]
Bump in version looks good.

**Comment by donal-tobin-sap** [comment]
New express version and 2 more vulnerabilities fixed overnight

**Review by devinea — APPROVED** [review]
security updates look good.

**Review by Klaus-Keller — APPROVED** [review]
Version bump looks good.

**Review by longieirl — APPROVED** [review]
Changes look good.

**Review by IainSAP — APPROVED** [review]

**Pull request merged**